test/test_meta.py

```diff
@@ -1086,6 +1097,20 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
 
         return expected
 
+
+def _handle_foreach_kwargs(func, args, kwargs):
+    if 'zero_size' in kwargs:
+        kwargs.pop('zero_size')
+    if 'disable_fastpath' in kwargs:
+        kwargs.pop('disable_fastpath')
+    if func in (torch._foreach_addcmul, torch._foreach_addcmul_, torch._foreach_addcdiv, torch._foreach_addcdiv_):
+        if 'values' in kwargs:
+            values = kwargs.pop("values")
+            if values is not None:
+                args = list(args)
+                args.append(values)
```

Comment:
Probably rather than doing this, we may want to have separate sample functions for these 4 that sample the correct inputs